idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -388,6 +436,9 @@ def run_chunk(self, part_suffix, input_files, chunk_count, lazy_run):
             project_id, sample_id = m.group(1), m.group(2)
         else:
             project_id, sample_id = '0', '0'
+
+        provisioning_model = 'SPOT'
+
         job_name = f"idseq-{deployment_environment}-{self.alignment_algorithm}-project-{project_id}-sample-{sample_id}-part-{chunk_id}"
         job_queue = f"idseq-{deployment_environment}-{self.alignment_algorithm}-{provisioning_model}-{index_dir_suffix}-{priority_name}"
```

Comment:
Ideally it would be best to keep the job queue names as parameters in the parameter store or environment variables set on the job. I recognize the difficulty of doing that before we retire the old dispatch infra, though, so I suggest filing that as a separate task under the "sfn/wdl operational improvements" epic.